src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace.cs

```diff
@@ -98,6 +100,8 @@ public TestWorkspace(ExportProvider? exportProvider = null, TestComposition? com
 
             _metadataAsSourceFileService = ExportProvider.GetExportedValues<IMetadataAsSourceFileService>().FirstOrDefault();
 
+            _refactorNotifyServices = ExportProvider.GetExports<IRefactorNotifyService>();
```

Comment:
Can we avoid fetching IRefactorNotifyService unless somebody calls TryApplyChanges? This code is fairly high traffic in unit tests so could have a non-trivial impact on unit test runtime.

Reply:
Changed to lazy so it's only fetched on use. Does mean that the ExportProvider will be captured, but I doubt that's a problem